tasks/run-local.sh

```diff
@@ -257,8 +259,8 @@ test_mock_pr() {
         # start mock GH server
         PYTHONPATH=. ./mock-github cockpit-project/bots \$SHA &
         GH_MOCK_PID=\$!
-        export GITHUB_API=http://127.0.0.7:8443
-        until curl --silent \$GITHUB_API; do sleep 0.1; done
+        export GITHUB_API=$GHAPI_URL_POD
+        until curl --silent --fail \$GITHUB_API/repos/cockpit-project/bots; do sleep 0.1; done
```

Comment:
I almost commented here that hardcoding this was bad, for example, but that's obviously non-sense in this case.

Sometimes I think it would be better to have named this repository something different.  I always get confused about things on the real GitHub (which should allow for generalization based on which fork is running the tests) vs. our mocked-up stuff.

This is officially a drive-by.